drivers/cc110x_legacy_csma/cc1100_phy.c

```diff
@@ -669,11 +669,9 @@ static void *cc1100_event_handler_function(void *arg)
                 rx_buffer_head = 0;
             }
 
-            eINT();
+            restoreIRQ(mask);
         }
 
-        dINT();
```

Comment:
It did not seem correct that interrupts were being disabled before calling `vtimer_set_msg`, is this removal correct?